test/test_simulation_execute.jl

```diff
@@ -129,10 +129,6 @@ end
                 binary_source_problem = PSI.ON,
                 affected_variables = [PSI.ACTIVE_POWER],
             ),
-            ("ED", :devices, :HydroEnergyReservoir) => IntegralLimitFF(
```

Comment:
Why are we removing these? I don't understand.

Reply:
Like I said the ramp was relaxed so the problem was feasible, with that enforced the problem becomes infeasible in the 2nd period, because IntegralLimitFF is forcing hydro output  to be zero in the 2nd period and you don't have enough to meet the balance constraint.

Reply:
Just realized I can add balance slacks to make the test pass,

Reply:
or better, you can adjust the data so it isn't infeasible and still does the correct thing. The solution of removing test coverage to make the tests pass isn't a good idea.

Reply:
ok, I'll change the data